mobile/library/common/engine_common.cc

```diff
@@ -100,10 +100,12 @@ EngineCommon::EngineCommon(std::shared_ptr<Envoy::OptionsImplBase> options) : op
       };
   // `set_new_handler` is false because the application using Envoy Mobile should decide how to
   // handle `new` memory allocation failures.
-  base_ = std::make_unique<StrippedMainBase>(
-      *options_, real_time_system_, default_listener_hooks_, prod_component_factory_,
-      std::make_unique<PlatformImpl>(), std::make_unique<Random::RandomGeneratorImpl>(), nullptr,
-      create_instance, /*set_new_handler=*/false);
+  std::unique_ptr<Random::RandomGeneratorImpl> random_generator =
```

Comment:
nit: this is a good use-case for `auto` as the type is obvious from context.

Reply:
done